core/src/main/resources/jenkins/security/apitoken/ApiTokenPropertyConfiguration/help-tokenGenerationOnCreationEnabled.html

```diff
@@ -0,0 +1,5 @@
+<div>
+    If this capability is disabled (recommended for security reasons), the users will be forced to use the new system of API token. <br/>
```

Comment:
nit: I would reword this blurb to: "This option causes a legacy API token to be generated automatically for newly created users. Because legacy tokens are \<strong\>deprecated\</strong\>, we recommend disabling it and having users instead generate new API tokens from the user configuration page as needed."